src/mlpack/core/optimizers/sgd/test_function.cpp

```diff
@@ -35,6 +41,33 @@ double SGDTestFunction::Evaluate(const arma::mat& coordinates, const size_t i)
   }
 }
 
+double SGDTestFunction::Evaluate(const arma::mat& coordinates,
+                                 const size_t begin,
+                                 const size_t batchSize) const
+{
+  double objective = 0;
+
+  for (int i = begin; i < begin + batchSize; i++)
+  {
+    switch (visitationOrder(i))
+    {
+      case 0:
+        objective -= std::exp(-std::abs(coordinates[0]));
+        break;
+
+      case 1:
+        objective += std::pow(coordinates[1], 2);
+        break;
+
+      case 2:
+        objective += std::pow(coordinates[2], 4) + \
```

Comment:
When wrapping a line, the next line should be tabbed twice from where the previous line began. For more information take a look at: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping. You can also remove the ``\`` at the end of the line.